event_lead_event/models/event.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import _
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class Event(models.Model):
+
+    _inherit = "event.event"
+
+    lead_event = fields.Boolean(
+        string='Lead Event',
+        index=True,
```

Comment:
required = True, default=False ?

Reply:
@grindtildeath If no default is given the field is already false. Since it is a boolean field it only has two states True and False so required either does nothing or our boolean field has to be true which is not what we want.

Reply:
definitely not `required` but `default=False` is much more explicit.